coverage/coverage-summary.json

```diff
@@ -1,31 +1,31 @@
-{"total": {"lines":{"total":802,"covered":590,"skipped":0,"pct":73.57},"statements":{"total":821,"covered":600,"skipped":0,"pct":73.08},"functions":{"total":181,"covered":129,"skipped":0,"pct":71.27},"branches":{"total":845,"covered":524,"skipped":0,"pct":62.01}}
+{"total": {"lines":{"total":809,"covered":279,"skipped":0,"pct":34.49},"statements":{"total":826,"covered":282,"skipped":0,"pct":34.14},"functions":{"total":181,"covered":54,"skipped":0,"pct":29.83},"branches":{"total":856,"covered":191,"skipped":0,"pct":22.31}}
```

Comment:
Did coverage drop by half? The heck happened here

Reply:
Hmmm thats super weird, I just checked my local coverage summary file and coverage looks like it went up, I'll re-push!